synapse/lib/view.py

```diff
@@ -28,15 +31,15 @@ class View(s_base.Base):
             core (Cortex):  The cortex that owns the view.
             node (HiveNode): The hive node containing the view info.
         '''
-        await s_base.Base.__anit__(self)
-
-        self.core = core
-
         self.node = node
         self.iden = node.name()
+        await s_hive.AuthGater.__anit__(self, core.auth)
+
+        self.core = core
```

Comment:
can we make this consistent with the layer constructor and move self.core above the __anit__ call and use self.core.auth ?

Reply:
ok